banyanmonitorgui/banyan_monitor_gui.py

```diff
@@ -290,11 +288,15 @@ def start_gui():
                         help="Publisher IP port")
     parser.add_argument("-s", dest="subscriber_port", default='43125',
                         help="Subscriber IP port")
-    parser.add_argument("-t", dest="topic_names", default="")
+    parser.add_argument("-t", dest="topic_names", default="",
+                        help="Separate topic names with commas")
```

Comment:
added a help line - took me a while to figure out that the topics needed to be comma delimited.